doc/src/learn/cryptography/sui-wallet-specs.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Sui Wallet Specifications - Keys and Addresses
+---
+
+Sui Wallet adheres to widely accepted wallet specifications in the cryptocurrency industry, including [BIP-32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) (and its variation, [SLIP-0010](https://github.com/satoshilabs/slips/blob/master/slip-0010.md)) and its variation [SLIP-0010](https://github.com/satoshilabs/slips/blob/master/slip-0010.md), [BIP-44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki), and [BIP-39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki), to facilitate key management for users. At present, Sui supports Ed25519, ECDSA Secp256k1, ECDSA Secp256r1, and [multisig](https://github.com/MystenLabs/sui/blob/6780cf62150893285f5db11f40f793fc440ec08b/doc/src/learn/cryptography/sui-multisig.md) for signed transactions.
+
+
+## Key Derivation Scheme
+
+Sui follows BIP-32 for managing wallets that support the ECDSA Secp256k1 signing scheme.
+
+BIP-32 defines the hierarchical deterministic wallet structure to logically associate a set of keys. Grouping keys in this manner reduces the overhead of keeping track of a large number of private keys for a user. This method also lets custodians issue distinct managed addresses for each user account under one source of control. Using BIP-32 decouples the private key derivation from the public key derivation, enabling the watch-only wallet use case, where a chain of public keys and its addresses can be derived, while the private key can be kept offline for signing.
```

Comment:
It's fine to use terms from the BIP32 terminology like non hardened public keys etc. this will easier imply why ed25519's SLIP0010 doesn't support that.